src/main/python/ttconv/scc/utils.py

```diff
@@ -44,3 +45,25 @@ def get_extent_from_dimensions(width: Union[int, Number], height: Union[int, Num
   width = LengthType(value=width, units=units)
 
   return ExtentType(height, width)
+
+
+def convert_cells_to_percentages(dimensions: Union[PositionType, ExtentType], cell_resolution: CellResolutionType) -> Union[
+  PositionType, ExtentType]:
+  """Converts dimensions from cell units to percentage units"""
+  if isinstance(dimensions, PositionType):
+    if dimensions.x.units is not LengthType.Units.c or dimensions.y.units is not LengthType.Units.c:
+      raise ValueError("Dimensions to convert must be in cell units")
+
+    x_percent = dimensions.x.value * 100 / cell_resolution.columns
+    y_percent = dimensions.y.value * 100 / cell_resolution.rows
```

Comment:
Percentages should be rounded to the nearest integer

Reply:
Fixed in 17ee6eb